src/main/java/ladder/domain/player/Players.java

```diff
@@ -0,0 +1,34 @@
+package ladder.domain.player;
+
+import ladder.exception.NeedMorePlayerException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Players {
+
+    private static final int MIN_PERSON_NUM = 1;
+
+    private final List<Player> playerList = new ArrayList<>();
+
+    public Players(String playerNames) {
+        String[] names = playerNames.split(",");
+
+        validate(names);
+
+        Arrays.stream(names).forEach(name -> playerList.add(new Player(name)));
+    }
+
+    private void validate(String[] names) {
+        if (names.length == MIN_PERSON_NUM) {
+            throw new NeedMorePlayerException("사다리 게임을 진행할 플레이어가 더 필요합니다");
+        }
+    }
+
+    public List<Player> getPlayers() {
+        return playerList;
```

Comment:
```suggestion
        return Collections.unmodifiableList(playerList);
```

변수 앞에 final을 사용해도 List는 참조 변수이기 때문에 List 안의 값은 수정 할 수 있어요.
이렇게 수정한다면 어떨까요? 🤔